main.py

```diff
@@ -1,11 +1,49 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
 
+    with open("players.json", "r") as file:
+        json_file = json.load(file)
+
+    for player in json_file:
+        info = json_file[player]
+        race = info.get("race")
+        guild = info.get("guild")
+        skills = info["race"]["skills"]
+
+        Race.objects.get_or_create(
+            name=race.get("name"),
+            description=race.get("description")
+        )
+        race_id = Race.objects.get_or_create(
+            name=race.get("name"), description=race.get("description")
+        )[0]
+
```

Comment:
Why do you call get_or_create twice?
You must search race only by name, you can pass description as `defaults`. Read more: https://docs.djangoproject.com/en/4.2/ref/models/querysets/#django.db.models.query.QuerySet.aget_or_create